api/services/call_log_service.py

```diff
@@ -113,17 +113,18 @@ def handle_parent_flow(self, jsonData):
         parent_flow_data = {}
         parent_flow_data["parent_flow_name"] = None
         parent_flow_data["parent_flow_run_uuid"] = None
-        if "parent" in jsonData and "flow" in jsonData["parent"]:
-            parent_flow = jsonData["parent"]["flow"]
-            parent_flow_data["parent_flow_name"] = parent_flow["name"]
-            missedcall_category_list = helpers.list_having_string(
-                parent_flow["name"], self.missedcall_flow_identifier
-            )
-            if missedcall_category_list:
-                """The call category is set to call back if missedcall flow has ran.
-                For that, the missed call flow name should contains string "missedcall"
-                """
-                self.call_category = models.CallLog.CallCategories.CALLBACK
+        if "parent" in jsonData and jsonData["parent"] is not None:
```

Comment:
This was the issue with empty  `jsonData["parent"]` on production. When jsonData["parent"] was empty, was throwing `TypeError: argument of type 'NoneType' is not iterable`